packages/cosmic-swingset/test/test-make.js

```diff
@@ -43,3 +68,92 @@ test('make and exec', async t => {
     }),
   );
 });
+
+const query = async args => {
+  const parts = [];
+  const run = new Promise((resolve, reject) => {
+    const cmd = spawn(
+      'bin/ag-chain-cosmos',
+      ['--chain-id=agoric', 'query', ...args],
+      { cwd: `${dirname}/..` },
+    );
+    cmd.stdout.on('data', chunk => parts.push(chunk));
+    cmd.on('exit', code => {
+      if (code !== 0) {
+        reject(Error(`exit ${code} from: query ${args}`));
+        return;
+      }
+      const txt = parts.join('');
+      if (txt.trim() === '') {
+        reject(Error(`empty output from: query ${args}`));
+        return;
+      }
+      const info = JSON.parse(txt);
+      resolve(info);
+    });
+  });
+  return run;
+};
+
+const waitForBlock = async height => {
+  const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+  for (;;) {
+    try {
+      const info = await query(['block']);
+      if (Number(info.block.header.height) >= height) {
+        return;
+      }
+    } catch (reason) {
+      console.warn(reason);
+      await delay(5_000);
+    }
+  }
+};
+
+test.serial('PSM bootstrap and provision', async t => {
+  // XXX coordinate with vbank/provision from golang
+  const pool = 'agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346';
+
+  const addrLine = await fs.promises.readFile(
+    './t1/8000/ag-cosmos-helper-address',
+    'utf-8',
+  );
+  const addr = addrLine.trim();
+
+  const show = args => {
+    t.log('make', ...args);
+    return args;
+  };
```

Comment:
this is a little surprising. I'd more expect logging `make` to be the concern of `runMake`. consider giving it an optional `log` argument that defaults to noop:
```
    await runMake(
      [`ACCT_ADDR=${addr}`, 'SOLO_COINS=123000000ubld', 'fund-acct'],
      t.log,
    );
```